src/components/IndividualArticle/IndividualArticle.jsx

```diff
@@ -15,6 +19,12 @@ function IndividualArticle() {
     });
   }, [article_id]);
 
+  useEffect(() => {
+    getCommentsById(article_id).then((comments) => {
+      setArticleComments(comments);
+    });
+  }, [article_id]);
```

Comment:
this useEffect could be in the commentlist component and make this single article component a bit less 'busy'.